modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,126 @@
+* Review the <<considerations,scope and limitations>>
+
+[[considerations]]
+== Scope and Limitations
+
+Before you migrate your client applications to Anypoint Platform, review the following scope and limitations: 
+
+* If the organization of the API that your application consumes has registered an external client provider (for example, PingFederate or Okta), review the section: <<external_client,Organizations with Registered External Client Providers>>.
+* Do not attempt to update the client ID and client secret of your application.
++
+This feature is not supported.
+* Because client IDs are unique in Anypoint Platform, do not reuse the same client ID in two different accounts.
++
+To avoid the duplication, specify unique client IDs for your accounts.
+* Do not use the same user to create more than 500 applications.
+* Do not create applications with duplicate names.
++
+Even if the identity provider (IdP) stores each applications, including duplicate applications, with different client ID and client secret credentials, issues can occur.
+*  Do not attempt to refresh the client ID unless you first delete and then re-register the client application.
+* If the client credentials you provide match the credentials of an existing organization ID on the client provider, the application is associated with that client.
++
+Conversely, if the client credentials that you provide do not match, the client application is created with no association to existing clients.
+
+
+[[migrate_with_internal]]
+== Migrate Client Applications for APIs with Anypoint Platform Client Providers
+
+To migrate your client application:
+
+. Open a text editor and paste the following commands:
++
+[source,text,linenums]
+----
+curl --location --request POST 'https://anypoint.mulesoft.com/exchange/api/v1/organizations/:orgId/applications' \
+--header 'Authorization: Bearer <omitted>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "clientId": <myClientId>,
+    "clientSecret": <myClientSecret>,
+    "grantTypes": [],
+    "name": <myApplicationName>,
+    "redirectUri": []
+}'
+----
+. Replace the `orgId` parameter with the ID of your master organization and the content of the body with the appropriate application <<properties,properties>>. 
+. Run this command at the Command prompt.
+
+[[migrate_with_external]]
+== Migrate Client Applications for APIs with External Client Providers
+
+When creating an application for an organization with registered external client providers, you can directly associate the application with the API that you want to consume, using that API instance ID (available from the API Manager UI).
+
+If you use a PingFederate external client provider or the Anypoint Platform client provider, you can include the client ID and client secret when referencing an API. Otherwise, for providers like Okta or Keycloak, which are registered using OpenID Dynamic registration, you cannot define client ID and client secret credentials.
```

Comment:
```suggestion
If you use a PingFederate external client provider or the Anypoint Platform client provider, you can include the client ID and client secret when referencing an API. Otherwise, for providers like Okta or Keycloak, which are registered using OpenID Dynamic Client Registration, you cannot define client ID and client secret credentials.
```

Reply:
Done! Thanks!